prog/weaver/main.go

```diff
@@ -497,6 +517,31 @@ func (nopPacketLogging) LogPacket(string, weave.PacketKey) {
 func (nopPacketLogging) LogForwardPacket(string, weave.ForwardPacketKey) {
 }
 
+func configureProxy(version string, defaultDockerHost string) *weaveproxy.Config {
+	proxyConfig := weaveproxy.Config{
+		Version:      version,
+		Image:        getenvOrDefault("EXEC_IMAGE", "weaveworks/weaveexec"),
+		DockerBridge: getenvOrDefault("DOCKER_BRIDGE", "docker0"),
+		DockerHost:   defaultDockerHost,
+	}
+	mflag.BoolVar(&proxyConfig.Disabled, []string{"-no-proxy"}, false, "instruct Weave not to start the Docker proxy")
```

Comment:
Sure, will do :+1: 
I did it this way cause I thought grouping may make sense.
Maybe we should just have all `mflag` instructions next to each others, but use separate `struct`s for proxy, router, etc. to separate concerns?